doc/ref/gappkg.xml

```diff
@@ -34,69 +34,67 @@ and then provide guidelines for writing a &GAP; package.
 <Section Label="Installing a GAP Package">
 <Heading>Installing a GAP Package</Heading>
 
-Before a  package can be  used it must  be installed. A standard installation
-of &GAP; already contains all &GAP; packages currently redistributed with 
-&GAP;. This set of packages is checked for compatibility
-with the system and other packages during release preparation. Most of
-the packages can be used immediately, but some of them may require further
-installation steps such as compilation or installing additional software
-to satisfy their dependencies. Most of the packages that require compilation
-can be compiled in a single step by changing to the <F>pkg</F> directory of
-your &GAP; installation and calling the <C>../bin/BuildPackages.sh</C> script.
+Before a  package can be  used it must  be installed. 
+A standard distribution of &GAP; already contains all the packages 
+currently available with &GAP;. 
+This set of packages has been checked for compatibility
+with the system and with each other during release preparation.  
+Most of the packages can be used immediately, but some of them may require further installation steps (see below). 
 <P/>
 Also, since &GAP; packages are released independently of the main &GAP; system,
-sometimes it may be useful to upgrade or install new packages between
+it may sometimes be useful to upgrade or install new packages between
 upgrades of your &GAP; installation, e.g. if a new version of a package adds
 new capabilities or bug fixes that you need.
 <P/>
 A package consists of a collection  of files within a single directory
 that must  be a subdirectory of  the <F>pkg</F>  directory   in one  of the
-&GAP; root directories, see <Ref Sect="GAP Root Directories"/>. (If you don't
-have access  to the <F>pkg</F>  directory in your main  &GAP; installation
-you can add private root directories as explained in that section.)
-<P/>
-Whenever  you  get from  somewhere  an  archive  of a  &GAP;  package
-it  should be  accompanied  with  a <F>README</F>  file  that explains  its
-installation.  Some packages  just  consist  of  &GAP;  code and  the
-installation  is  done  by  unpacking   the  archive  in  one  of  the
-places  described above.  There are  also packages  that need  further
-installation steps,  there may be  for example some  external programs
-which  have  to  be  compiled  (this is  often  done  by  just  saying
-<C>./configure; make</C>  inside the unpacked package  directory, but check
-the individual <F>README</F> files). Note that if you use Windows you may not 
-be able to use some or all external binaries.
+&GAP; root directories (see <Ref Sect="GAP Root Directories"/>). 
+If you don't have access  to the <F>pkg</F>  directory in your main  &GAP; installation you can add private root directories as explained in that section. 
+<P/>
+Whenever you download an  archive  of a  &GAP;  package, it will contain a 
```

Comment:
Of course, you can clone a repository instead of downloading, and a package which is mot redistributed with GAP may accidentally not have README (or README.txt or README.md) at all...